gapis/gfxapi/templates/api.go.tmpl

```diff
@@ -127,6 +127,11 @@
     {{$to := Macro "Go.Type" $.To}}
 
     type {{$.Name}} {{$to}}
+    {{if (GetAnnotation $ "dispatchHandle")}}
```

Comment:
I see, so I should have a template to check whether the underlying type is int, uint, and char. I will do that.